SPECS/curl.spec

```
@@ -0,0 +1,356 @@
+        --disable-tls-srp               \
+        --disable-websockets            \
+        --disable-alt-svc               \
+        --disable-tls-srp               \
+        --without-brotli                \
+        --without-libpsl                \
+        --with-${libssh}
+)
+
+# avoid using rpath
+sed -e 's/^runpath_var=.*/runpath_var=/' \
+    -e 's/^hardcode_libdir_flag_spec=".*"$/hardcode_libdir_flag_spec=""/' \
+    -i build-full/libtool
+
+%make_build V=1 -C build-full
+
+%check
+# compile upstream test-cases
+%make_build V=1 -C build-full/tests
+
+# relax crypto policy for the test-suite to make it pass again (#1610888)
+export OPENSSL_SYSTEM_CIPHERS_OVERRIDE=XXX
+export OPENSSL_CONF=
+
+# make runtests.pl work for out-of-tree builds
+export srcdir=../../tests
+
+# prevent valgrind from being extremely slow (#1662656)
+# https://fedoraproject.org/wiki/Changes/DebuginfodByDefault
+unset DEBUGINFOD_URLS
+
+# run the upstream test-suite
+for size in full; do (
+    cd build-${size}
+
+    # we have to override LD_LIBRARY_PATH because we eliminated rpath
+    export LD_LIBRARY_PATH="${PWD}/lib/.libs"
+
+    cd tests
+    perl -I../../tests ../../tests/runtests.pl -a -p -v '!flaky'
+)
+done
+
+
+%install
+# install libcurl.m4
+install -d $RPM_BUILD_ROOT%{_datadir}/aclocal
+install -m 644 docs/libcurl/libcurl.m4 $RPM_BUILD_ROOT%{_datadir}/aclocal
+
+# install the executable and library that will be packaged as curl and libcurl
+cd build-full
+%make_install
+
+# install zsh completion for curl
+# (we have to override LD_LIBRARY_PATH because we eliminated rpath)
+LD_LIBRARY_PATH="$RPM_BUILD_ROOT%{_libdir}:$LD_LIBRARY_PATH" \
+    %make_install -C scripts
+
+# do not install /usr/share/fish/completions/curl.fish which is also installed
+# by fish-3.0.2-1.module_f31+3716+57207597 and would trigger a conflict
+rm -rf ${RPM_BUILD_ROOT}%{_datadir}/fish
+
+rm -f ${RPM_BUILD_ROOT}%{_libdir}/libcurl.la
+
+# Don't install man for mk-ca-bundle it's upstream bug
+# should be fixed in next release https://github.com/curl/curl/pull/12843
+rm -f ${RPM_BUILD_ROOT}%{_mandir}/man1/mk-ca-bundle.1*
+
+/sbin/ldconfig -n libcurl
+
+%files
+%doc CHANGES
+%doc README
+%doc docs/BUGS.md
+%doc docs/FAQ
+%doc docs/FEATURES.md
+%doc docs/TODO
+%doc docs/TheArtOfHttpScripting.md
+%{_bindir}/curl
+%{_mandir}/man1/curl.1*
+%{_datadir}/zsh
+
+%files -n libcurl
+%license COPYING
+%{_libdir}/libcurl.so.4
+%{_libdir}/libcurl.so.4.[0-9].[0-9]
+
+%files -n libcurl-devel
+%doc docs/examples/*.c docs/examples/Makefile.example docs/INTERNALS.md
+%doc docs/CONTRIBUTE.md docs/libcurl/ABI.md
+%{_bindir}/curl-config*
+%{_includedir}/curl
+%{_libdir}/*.so
+%{_libdir}/pkgconfig/*.pc
+%{_mandir}/man1/curl-config.1*
+%{_mandir}/man3/*
+%{_datadir}/aclocal/libcurl.m4
+
+%changelog
+* Wed May 22 2024 Gael Duperrey <gduperrey@vates.tech> - 8.6.0-2.1
+- Synced from XS82ECU1063
+- Removed xenserver-specific test of %dist to determine buildrequires
```

Incomplete. It's not just for the buildrequires.

And you didn't escape %dist.

```suggestion
- Removed xenserver-specific test of the dist macro
```